examples/restaurant_demo/README.md

```diff
@@ -0,0 +1,260 @@
+# Example using the Differential Privacy Library PyDP
+
+## Restaurant
+
+Imagine a fictional restaurant owner named Alice who would like to share
+business statistics with her visitors. Alice knows when visitors enter the
+restaurant and how much time and money they spend there. To ensure that
+visitors' privacy is preserved, Alice decides to use the Differential Privacy
+library.
+
+## Installation
+To install PyDP, use the PyPI package manager:
+
+pip install python-dp
+(If you have pip3 separately for Python 3.x, use pip3 install python-dp.)
+
+Navigate to `PyDP/examples/restaurant_demo` folder. Execute `Python restaurant.py`
+
+The output will display Private and Non-Private counts for:
```

Comment:
mainly 3 scenarios Count visits by hour of day,  Count visits by day of week,  Sum-up revenue per day of the week

Count visits by day of week -- has two cases - max_contribution = 7 or 3 days/week
Sum-up revenue per day of the week -- same but with upper bound set to 65  --  max_contribution= 4 days/week

Reply:
Corrected